packages/mgt-components/src/components/mgt-todo/mgt-todo.ts

```diff
@@ -376,7 +366,9 @@ export class MgtTodo extends MgtTasksBase {
     if (this.hasTemplate('task-details')) {
       taskDetailsTemplate = this.renderTemplate('task-details', context, `task-details-${task.id}`);
     } else {
-      const dateClass = { dark: this._isDarkMode, date: true };
+      const hideDate = task.dueDateTime || this._updatingTask ? false : true;
```

Comment:
At the moment to have the date input become visible if there's no date it's necessary to alter the value of the task title input.

Can we capture that a task is in edit mode when an element in the row gets focus?